flask_security/core.py

```diff
@@ -88,13 +89,14 @@
     'CHANGE_SALT': 'change-salt',
     'REMEMBER_SALT': 'remember-salt',
     'DEFAULT_REMEMBER_ME': False,
-    'DEFAULT_HTTP_AUTH_REALM': 'Login Required',
-    'EMAIL_SUBJECT_REGISTER': 'Welcome',
-    'EMAIL_SUBJECT_CONFIRM': 'Please confirm your email',
-    'EMAIL_SUBJECT_PASSWORDLESS': 'Login instructions',
-    'EMAIL_SUBJECT_PASSWORD_NOTICE': 'Your password has been reset',
-    'EMAIL_SUBJECT_PASSWORD_CHANGE_NOTICE': 'Your password has been changed',
-    'EMAIL_SUBJECT_PASSWORD_RESET': 'Password reset instructions',
+    'DEFAULT_HTTP_AUTH_REALM': _('Login Required'),
```

Comment:
> `lazy_gettexts` are only evaluated when used ...

This is a problem for configuration strings provided manually as `str`. They would not be translated when used.